code/src/java/pcgen/io/exporttoken/AbilityToken.java

```diff
@@ -420,7 +411,8 @@ protected String getTokenForCategory(String tokenSource,
 	 * @return True if it matches one of the types else false
 	 */
 	static boolean abilityMatchesType(String abilityType,
-		Ability aAbility, List<String> types, List<String> negate)
+	                                  Ability aAbility, Collection<String> types,
```

Comment:
Similar to another PR, we need to get consistent on these wrapping rules.  This doesn't match how the standards file wraps